config/typescript-axios.yaml

```diff
@@ -0,0 +1,7 @@
+generatorName: typescript-axios
+outputDir: /local/src/typescript/mit-open-api-axios
+inputSpec: /open/openapi.yaml
```

Comment:
I just started a release on MIT Open after which this will be `/open/openapi/v1/openapi_spec.yaml`